packages/mountable/src/dune/hooks/useFetchFileContent.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { renderHook, waitFor } from "@testing-library/react";
+import { useFetchFileContent } from "./useFetchFileContent";
+import { type Credentials } from "../utils/fileUtils";
+
+const mockRetrieveFileMetadata = jest.fn() as jest.MockedFunction<
+  (externalId: string, credentials: Credentials) => Promise<any>
+>;
+const mockGetFileDownloadUrl = jest.fn() as jest.MockedFunction<
+  (
+    externalId: string,
+    credentials: Credentials,
+  ) => Promise<{ downloadUrl: string }>
+>;
+
+describe("useFetchFileContent", () => {
+  const mockFetch = jest.fn() as jest.MockedFunction<typeof fetch>;
+  const mockCredentials: Credentials = {
+    token: "test-token",
+    project: "test-project",
+    baseUrl: "https://api.cognitedata.test",
+  };
+
+  beforeEach(() => {
+    global.fetch = mockFetch;
```

Comment:
```suggestion
    jest.spyOn(global, "fetch").mockImplementation(mockFetch);
```

and add a `afterEach(() => { jest.restorAlLMocks(); })`

Reply:
Fixed in 96e8b815ab4367eba735c0b07c63289ac0b3f5f8